src/components/pages/state/charts/index.js

```diff
@@ -14,11 +26,51 @@ const dailyAverage = (history, field, range = 7) => {
   return average
 }
 
-export default ({ history }) => (
-  <div>
-    Charts go here. Daily avereage of positive:{' '}
-    {dailyAverage(history, 'positive').map(average => (
-      <>{average} </>
-    ))}
-  </div>
-)
+// must be of format:
+//  {
+//   date: Date
+//   value: number
+// }
+const getDataForField = (data, field) => {
+  return data.map(d => ({
+    date: parseDate(d.date),
+    value: d[field],
+  }))
+}
+
+// we need a combined bar + line chart
+
+export default ({ history }) => {
+  const data = [...history].sort((a, b) => a.date - b.date)
```

Comment:
This sort is being applied to the parent history array. Let's clone it so it won't also sort the history table.